### PR TITLE
Removes broken support e-mail address in the README.md, and improve the support section

### PR DESCRIPTION
**Pull request by seharbat00l**
Maintainer merge checklist
* [x] Title is descriptive/clear for inclusion in release notes.
* [x] Applied a `Component: xxx` label.
* [ ] Applied the `api-deprecation` or `api-break` label.
* [ ] Applied the `release-highlight` label to be highlighted in release notes.
* [x] Added to the milestone version it was merged into.
* [ ] **Unittests** are included in PR.
* [ ] Properly documented, including `versionadded`, `versionchanged` as needed.

The support section in Readme.md file has been fixed🚀 issue #8409 (broken support mail)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,15 +44,11 @@ Kivy ships with many examples which can be found in the `examples` folder.
 Support
 -------
 
-If you need assistance, you can ask for help on our mailing list:
+If you need help, please feel free to seek assistance through our support channels:
 
-* User Group : https://groups.google.com/group/kivy-users
-* Email      : kivy-users@googlegroups.com
+Join the  [Kivy Users Google Group](https://groups.google.com/group/kivy-users).
+Participate in discussions on [Discord](https://chat.kivy.org).
```
Comment (p0lygun) [inline README.md:50]: Nice work 👍 , while we are at it, i think its ok if we also add link to our all new [discussion](https://github.com/orgs/kivy/discussions) threads here on GH

```suggestion
- Join the [Kivy Users Google Group](https://groups.google.com/group/kivy-users).
- Ask questions using [discussions](https://github.com/orgs/kivy/discussions/new?category=q-a) on github
- Use support-channels on our official [Discord](https://chat.kivy.org) server.  
```

Comment (seharbat00l) [reply README.md:50]: Sure

Comment (p0lygun) [reply README.md:50]: @seharbat00l updates on this?

File: README.md [hunk 2]
```diff
@@ -46,8 +46,9 @@ Support
 
 If you need help, please feel free to seek assistance through our support channels:
 
-Join the  [Kivy Users Google Group](https://groups.google.com/group/kivy-users).
-Participate in discussions on [Discord](https://chat.kivy.org).
+Join the [Kivy Users Google Group](https://groups.google.com/group/kivy-users).
```
Comment (p0lygun) [inline README.md:49]:
```suggestion
- Join the [Kivy Users Google Group](https://groups.google.com/group/kivy-users).
```
missing bullet point :)

**Review by p0lygun — CHANGES_REQUESTED** [review]

**Review by p0lygun — APPROVED** [review]
@misl6 can we get this approved :)

**Review by misl6 — APPROVED** [review]
LGTM. Thank you!

**Pull request merged**